GREEDY/DataManagers/IGraphManager.cs

```diff
@@ -6,5 +6,10 @@ namespace GREEDY.DataManagers
     public interface IGraphManager
     {
         List<GraphData> GetDataForGraphs(string username, int? time);
+        List<AverageStorePriceGraphData> GetDataForAverageStorePriceGraph(string username, int? time);
```

Comment:
what is the purpose of the "time" variables?

Reply:
Honestly i don't know. This was in the example graph created by Marius. I had a lot of difficulty understanding what was needed and what was not since i have no idea how back end communicated with the front end.

Reply:
Time is basically when you have button in the front end for what time periods do you want the graph data to be displayed. This is a feature for the future maybe